tests/apollo/util/skvbc_history_tracker.py

```diff
@@ -573,7 +588,9 @@ def _match_filled_blocks(self):
                     break
                 else:
                     unmatched.append(req)
-            if not success:
+            # Don't raise PhantomBlockError for blocks created before the
+            # instantiation of the tracker
+            if not success and block_id > self.initial_block_id:
```

Comment:
This also feels wrong. We shouldn't need to check blocks prior to self.initial_block_id.